### PR TITLE
[14.0][MIG] product_multi_company

### PR DESCRIPTION
**Pull request by rousseldenis**
Includes:

- [x] #270

### CODE REVIEW

File: base_multi_company/hooks.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    :param: rule_ref: XML-ID of the security rule to change.
+    """
+    rule = env.ref(rule_ref)
+    if not rule:  # safeguard if it's deleted
+        return
+    rule.write(
+        {
+            "active": True,
+            "domain_force": (
+                "['|', ('company_ids', '=', False), ('company_ids', "
```
Comment (rousseldenis) [inline base_multi_company/hooks.py]: @LoisRForgeFlow Have you ever experienced issues with `company_ids = False `domain ?

I fixed it with `company_ids = []`

Comment (rousseldenis) [reply base_multi_company/hooks.py]: > @LoisRForgeFlow Have you ever experienced issues with `company_ids = False `domain ?
> 
> I fixed it with `company_ids = []`

Moreover, 'in' [] is the correct one.

**Review by james-willdooit — APPROVED** [review]
LGTM

**Review by GSLabIt — APPROVED** [review]
LGTM

**Comment by GSLabIt** [comment]
@rousseldenis any chance to merge?

**Comment by rousseldenis** [comment]
> @rousseldenis any chance to merge?

I have no rights on this. But we need base_multi_company approved first. Could you review it ?

**Comment by elvise** [comment]
@rousseldenis ready to merge ? :)

**Comment by matteoopenf** [comment]
@OCA/community-maintainers  I think is ready to merge

**Review by matteoopenf — APPROVED** [review]

**Comment by rousseldenis** [comment]
Rebased

**Comment by matteoopenf** [comment]
> Rebased

is ready again  to merge

**Comment by matteoopenf** [comment]
@OCA/intercompany-maintainers

**Comment by matteoopenf** [comment]
Hi @pedrobaeza, is it ok for you this pull request?
Thanks a lot

**Comment by pedrobaeza** [comment]
I can't say as I have not reviewed it, but based on existing reviews:

/ocabot migration product_multi_company
/ocabot merge nobump

**Comment by matteoopenf** [comment]
> I can't say as I have not reviewed it, but based on existing reviews:
> 
> /ocabot migration product_multi_company /ocabot merge nobump

Many thanks

**Pull request merged**